store/store.go

```diff
@@ -769,6 +769,10 @@ func (s *Store) Restore(rc io.ReadCloser) error {
 	}()
 }
 
+func (s *Store) RegisterObserver(o *raft.Observer) {
```

Comment:
Can you add a GoDoc string? E.g. `RegisterObserver registers an observer that ....`